http4k-client/websocket/src/test/kotlin/org/http4k/client/WebsocketClientNonBlockingTest.kt

```diff
@@ -1,10 +1,10 @@
 package org.http4k.client
 
-import org.http4k.server.Jetty
+import org.http4k.server.Undertow
 import org.http4k.websocket.NonBlockingWebsocketClientContract
 
 class WebsocketClientNonBlockingTest : NonBlockingWebsocketClientContract(
-    serverConfig = Jetty(0),
+    serverConfig = Undertow(0),
```

Comment:
not that I mind - but curious about why this was changed from Jetty? :)

Reply:
Oh that was a mistake. I wanted to use something else than Jetty for the OkHttp client, but I tried it out first with Java WS client and forgot to change it back. I'll do that now.

Reply:
Funny thing I just found by using Jetty server in the OkHttp client tests - it seems that Jetty sends an empty binary WS message after each text message, but Undertow does not. I have no idea why it would do that, maybe that is an issue on the Jetty side or in the http4k specific code that plugs in the WS functionality (I doubt it, that code looks simple enough).
I'll stick with using Undertow for the OkHttp tests for now.